scipy/stats/_continuous_distns.py

```diff
@@ -3261,6 +3262,313 @@ def _ppf(self, q, k, s):
 mielke = mielke_gen(a=0.0, name='mielke')
 
 
+class kappa4_gen(rv_continuous):
+    """Kappa 4 parameter distribution.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for kappa4 is::
+
+        kappa4.pdf(x, h, k) = (1.0 - k*x)**(1.0/k - 1.0)*
+                              (1.0 - h*(1.0 - k*x)**(1.0/k))**(1.0/h-1)
+
+    if ``h`` and ``k`` are not equal to 0.
+
+    If ``h`` or ``k`` are zero then the pdf can be simplified:
+
+    h = 0 and k != 0::
+
+        kappa4.pdf(x, h, k) = (1.0 - k*x)**(1.0/k - 1.0)*
+                              exp(-(1.0 - k*x)**(1.0/k))
+
+    h != 0 and k = 0::
+
+        kappa4.pdf(x, h, k) = exp(-x)*(1.0 - h*exp(-x))**(1.0/h - 1.0)
+
+    h = 0 and k = 0::
+
+        kappa4.pdf(x, h, k) = exp(-x)*exp(-exp(-x))
+
+    kappa4 takes ``h`` and ``k`` as shape parameters, skew and
+    kurtosis respectively.
```

Comment:
I'd say whether that statement implies a different definition of skew/kurtosis, or is it something those authors might want to write an erratum for, it's up to them. As far as I can see, this is not correct, and should not be included as written.